tree/ntuple/v7/src/RField.cxx

```diff
@@ -2191,6 +2188,93 @@ void ROOT::Experimental::RArrayField::AcceptVisitor(Detail::RFieldVisitor &visit
    visitor.VisitArrayField(*this);
 }
 
+//------------------------------------------------------------------------------
+// RArrayAsRVecField
+
+ROOT::Experimental::RArrayAsRVecField::RArrayAsRVecField(
+   std::string_view fieldName, std::unique_ptr<ROOT::Experimental::Detail::RFieldBase> itemField,
+   std::size_t arrayLength)
+   : ROOT::Experimental::Detail::RFieldBase(fieldName, "ROOT::VecOps::RVec<" + itemField->GetType() + ">",
+                                            ENTupleStructure::kCollection, false /* isSimple */, arrayLength),
```

Comment:
I absolutely agree with @jblomer.  Also, FWIW, (it does not matter here because this field is read-only), but note that getting the repetition level right is _crucial_ for deferred (i.e. late-added) fields to work!

Reply:
Done in latest commits